### ISSUE TITLE
Consent templates loaded from the wrong location

### ISSUE THREAD

**Issue by turt2live** [matrix-org/synapse#4501]
Synapse version: `develop` at time of writing.

`homeserver.yaml` bits:
```yaml
user_consent:
  template_dir: ./res/templates/privacy
  version: 1.0
  server_notice_content:
    msgtype: m.text
    body: >-
      To continue using this homeserver you must review and agree to the
      terms and conditions at %(consent_uri)s
  send_server_notice_to_guests: false
  block_events_error: >-
    To continue using this homeserver you must review and agree to the
    terms and conditions at %(consent_uri)s
  require_at_registration: true
  policy_name: "Terms and Conditions"

```

Error:
```
2019-01-28 21:20:04,020 - twisted - 242 - ERROR -  - ConfigError: Could not find template directory '/res/templates/privacy'
```

It shouldn't be trying to load the absolute path at all with this configuration.

See also #4500

**Comment by richvdh** [comment]
this sounds like a regression which we need to fix before the release this week

**Comment by richvdh** [comment]
This was introduced by https://github.com/matrix-org/synapse/pull/4384/files#diff-01aac508ef38061957e6fb961582e5ceL364; fixed by #4512.
